external/storm-kafka/src/jvm/storm/kafka/trident/TridentKafkaState.java

```diff
@@ -76,24 +77,41 @@ public void prepare(Map stormConf) {
         producer = new Producer(config);
     }
 
-    public void updateState(List<TridentTuple> tuples, TridentCollector collector) {
-        String topic = null;
-        for (TridentTuple tuple : tuples) {
-            try {
-                topic = topicSelector.getTopic(tuple);
+    @SuppressWarnings("rawtypes")
```

Comment:
@abellina, Thanks for the comment. I have fixed it now. let me know if any queries.